scripts/Diffraction/isis_powder/polaris_routines/polaris_algs.py

```diff
@@ -9,7 +9,7 @@
 
 
 def calculate_absorb_corrections(ws_to_correct, multiple_scattering):
-    mantid.MaskDetectors(ws_to_correct, SpectraList=list(range(0, 55)))
+    mantid.MaskDetectors(ws_to_correct, SpectraList=list(range(1, 55)))
```

Comment:
This is correct. I had already correctly taken into account the fact the last invalid spectra was detector 54 however failed to notice they start a 1 not 0 originally.